keel-docker/src/test/kotlin/com/netflix/spinnaker/keel/docker/SampleDockerImageResolverTests.kt

```diff
@@ -58,13 +58,20 @@ class SampleDockerImageResolverTests : JUnit5Minutests {
     ),
     account = "test"
   )
+
+  private val nullable = SampleSpecWithContainer(
+    account = "test"
+  )
+
   val versionedContainerResource = generateResource(oldStyleSpec)
 
   val referenceResource = generateResource(referenceSpec)
   val versionedDeliveryConfig = generateDeliveryConfig(versionedContainerResource, artifact)
-
   val referenceDeliveryConfig = generateDeliveryConfig(referenceResource, artifact)
 
+  private val nullableResource = generateResource(nullable)
+  private val nullableDeliveryConfig = generateDeliveryConfig(nullableResource, artifact)
```

Comment:
Suggestion for clarity:
```suggestion
  private val nullContainerDeliveryConfig = generateDeliveryConfig(nullableResource, artifact)
```